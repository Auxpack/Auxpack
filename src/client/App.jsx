import React from 'react';
import MainContainer from './containers/MainContainer.jsx'
import '../assets/css/styles.scss';
const App = (props) => {
    return (
        <MainContainer />
    );
}

export default App;