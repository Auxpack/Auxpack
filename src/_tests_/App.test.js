import App from '../client/App.jsx';
// see setupTests.js for config

describe('Examining the syntax of Jest tests', () => {
   
  it('sums numbers', () => {
      expect(1 + 2).toEqual(3);
      expect(2 + 2).toEqual(4);
   });
});

