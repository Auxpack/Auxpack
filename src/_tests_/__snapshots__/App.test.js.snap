// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot 1`] = `
<MainContainer
  id="main"
/>
`;
